### PR TITLE
DateTime metadata for Booking

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: datamodel/initdb.d/09_metadata.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+-- Assumes the PSQL client
```
Comment (HenrikHL) [inline datamodel/initdb.d/09_metadata.sql:1]: Why is it called 09? I thought 07 was the next slot...

Comment (mp-assco) [reply datamodel/initdb.d/09_metadata.sql:1]: Derp, must've mistyped...

Comment (mp-assco) [reply datamodel/initdb.d/09_metadata.sql]: @HenrikHL fixed

File: datamodel/initdb.d/09_metadata.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+\set ON_ERROR_STOP true
+\connect dcsa_openapi
+
+-- Currently an implementation detail for the reference implementation but one
+-- we would like to remove eventually.
+BEGIN;
+
+-- Metadata for Booking table to avoid having to query shipmentEvent for date_times necessary for BookingResponseTO
+ALTER TABLE dcsa_im_v3_0.booking
+    ADD COLUMN IF NOT EXISTS event_created_date_time timestamp with time zone NOT NULL;
```
Comment (HenrikHL) [inline datamodel/initdb.d/09_metadata.sql]: I don't think we should call it `event_`created_date_time. Can we drop that part?

File: datamodel/initdb.d/09_metadata.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ALTER TABLE dcsa_im_v3_0.booking
+    ADD COLUMN IF NOT EXISTS event_date_time timestamp with time zone NOT NULL;
```
Comment (HenrikHL) [inline datamodel/initdb.d/09_metadata.sql]: same

File: datamodel/initdb.d/09_metadata.sql (hunk 2 above)
Comment (mp-assco) [reply datamodel/initdb.d/09_metadata.sql]: @HenrikHL Should I rename it to `updated_date_time` and `created_date_time`?

File: datamodel/initdb.d/07_metadata.sql [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+\set ON_ERROR_STOP true
+\connect dcsa_openapi
+
+-- Currently an implementation detail for the reference implementation but one
+-- we would like to remove eventually.
+BEGIN;
+
+-- Metadata for Booking table to avoid having to query shipmentEvent for date_times necessary for BookingResponseTO
+ALTER TABLE dcsa_im_v3_0.booking
+    ADD COLUMN IF NOT EXISTS event_created_date_time timestamp with time zone NOT NULL;
```
Comment (pimg) [inline datamodel/initdb.d/07_metadata.sql:11]: I would suggest created_date_time + updated_date_time

**Comment by nt-as** [comment]
Can we merge the new 07 script into 06 if it is something we want to remove again later?  Otherwise, we would need to rename all the other scripts (07 is already taken under testdata).

**Review by pimg — CHANGES_REQUESTED** [review]
I find the names of the columns a bit confusing since they apply to the booking and not an event

**Comment by mp-assco** [comment]
> I find the names of the columns a bit confusing since they apply to the booking and not an event

@pimg updated names

**Review by pimg — APPROVED** [review]

**Review by nt-as — APPROVED** [review]
LGTM

**Review by pimg — CHANGES_REQUESTED** [review]
filename is still 07 this needs to be 06

**Review by pimg — APPROVED** [review]

**Pull request merged**